pyproject.toml

```diff
@@ -31,7 +31,7 @@ dependencies = [
     "pika >= 1.2.0",
     "dataset",
     "pymongo > 3.0",
-    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@main",
+    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@fix/issue_controller_161",
```

Comment:
Heads-UP: temporary change to be able to validate the pipeline. It will be reverted before merging